### PR TITLE
docs: update angular list

### PR DESCRIPTION
**Pull request by sympatheticmoose**
**What is this feature?**

[Add a brief description of what the feature or update does.]

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/developers/angular_deprecation/angular-plugins.md [hunk 1]
```diff
@@ -121,6 +121,10 @@ Latest Version: v0.0.6 | Signature: Community | Last Updated: 2021
 
 > **Migration available - plugin superseded:** The original plugin only claims support for Grafana v4.x.x, it was replaced with a [new plugin](https://grafana.com/grafana/plugins/grafadruid-druid-datasource/) based on React.
 
+### [Cognite Data Fusion](https://grafana.com/grafana/plugins/cognitedata-datasource/)
+
+Latest Version: 3.0.0 | Signature: Commercial | Last Updated: 2023
```
Comment (xnyo) [inline docs/sources/developers/angular_deprecation/angular-plugins.md:126]: Latest version is 3.1.0:

https://grafana.com/grafana/plugins/cognitedata-datasource/

immagine

```suggestion
Latest Version: 3.1.0 | Signature: Commercial | Last Updated: 2023
```

**Review by xnyo — COMMENTED** [review]
Thank you! :pray: 

Only one small change:

**Comment by sympatheticmoose** [comment]
@xnyo done, good spot!

**Review by xnyo — APPROVED** [review]
LGTM!

**Pull request merged**